bridge/standard/pkg/node/node.go

```diff
@@ -243,9 +243,18 @@ func (n *Node) createGatewayContract(
 		return fmt.Errorf("failed to parse contract ABI: %w", err)
 	}
 
+	wrappedClient, err := ethwrapper.NewClient(
+		logger.With("component", fmt.Sprintf("%s/ethwrapper", component)),
+		[]string{rpcURL},
+		ethwrapper.EthClientWithBlockNumberDrift(2*32), // 2 epochs
```

Comment:
By using `ethwrapper.EthClientWithBlockNumberDrift`, we're implicitly using `maxRetries` = 0 correct? Is this intentional for the relayer? From my perspective we may want at least a couple retries in the scenario of reorg